src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -176,8 +180,12 @@ open class ComposeTask : Zip(), AemTask {
     fun includeBundles(project: Project) {
         dependProject(project)
 
+        val extraProperties = project.extensions.extraProperties
+        val bundlePath: String = if (extraProperties == null || !extraProperties.has("bundlePath")) config.bundlePath
+        else extraProperties.get("bundlePath") as String
+
         bundleCollectors += {
-            JarCollector(project).all.toList()
+            Pair(bundlePath, JarCollector(project).all.toList())
```

Comment:
hmmm if you want to include project but have it located in different place, it just wasn't designed in that way :) I will think a little bit about edge cases and our development workflow issues that can coincide and tell you then.